GitUI/NBugReports/BugReportForm.Designer.cs

```diff
@@ -76,242 +88,303 @@ private void InitializeComponent()
+            this.dateTimeTextBox.TabIndex = 7;
             // 
             // errorDescriptionLabel
             // 
             this.errorDescriptionLabel.AutoSize = true;
-            this.errorDescriptionLabel.Location = new System.Drawing.Point(10, 242);
+            this.tpnlEnvInfo.SetColumnSpan(this.errorDescriptionLabel, 4);
+            this.errorDescriptionLabel.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.errorDescriptionLabel.Location = new System.Drawing.Point(11, 94);
             this.errorDescriptionLabel.Name = "errorDescriptionLabel";
-            this.errorDescriptionLabel.Size = new System.Drawing.Size(315, 13);
-            this.errorDescriptionLabel.TabIndex = 14;
-            this.errorDescriptionLabel.Text = "Please add a brief description of how we can reproduce the error:";
+            this.errorDescriptionLabel.Size = new System.Drawing.Size(512, 13);
+            this.errorDescriptionLabel.TabIndex = 10;
+            this.errorDescriptionLabel.Text = "Please add a brief description (in English) of how we can reproduce the error:";
+            this.errorDescriptionLabel.TextAlign = System.Drawing.ContentAlignment.MiddleLeft;
             // 
-            // clrTextBox
+            // applicationTextBox
             // 
-            this.clrTextBox.BackColor = System.Drawing.SystemColors.ControlLightLight;
-            this.clrTextBox.Location = new System.Drawing.Point(307, 208);
-            this.clrTextBox.Name = "clrTextBox";
-            this.clrTextBox.ReadOnly = true;
-            this.clrTextBox.Size = new System.Drawing.Size(145, 20);
-            this.clrTextBox.TabIndex = 13;
+            this.applicationTextBox.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.applicationTextBox.Location = new System.Drawing.Point(104, 37);
+            this.applicationTextBox.MaxLength = 100;
+            this.applicationTextBox.Name = "applicationTextBox";
+            this.applicationTextBox.ReadOnly = true;
+            this.applicationTextBox.Size = new System.Drawing.Size(160, 20);
+            this.applicationTextBox.TabIndex = 3;
             // 
-            // clrLabel
+            // targetSiteLabel
             // 
-            this.clrLabel.AutoSize = true;
-            this.clrLabel.Location = new System.Drawing.Point(259, 210);
-            this.clrLabel.Name = "clrLabel";
-            this.clrLabel.Size = new System.Drawing.Size(31, 13);
-            this.clrLabel.TabIndex = 12;
-            this.clrLabel.Text = "CLR:";
+            this.targetSiteLabel.AutoSize = true;
+            this.targetSiteLabel.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.targetSiteLabel.Location = new System.Drawing.Point(11, 8);
+            this.targetSiteLabel.Name = "targetSiteLabel";
+            this.targetSiteLabel.Size = new System.Drawing.Size(87, 26);
+            this.targetSiteLabel.TabIndex = 0;
+            this.targetSiteLabel.Text = "Target Site:";
```

Comment:
What is this?

Reply:
![image](https://user-images.githubusercontent.com/4403806/59557048-db9fba00-9012-11e9-8308-fa8928d591de.png)